PickMeUp/PickMeUp/ViewControllers/DM/PMDisplayConversationsViewController.m

```diff
@@ -34,54 +36,26 @@ - (void)viewDidLoad {
     [self.searchField addTarget:self
                   action:@selector(_textFieldDidChange)
         forControlEvents:UIControlEventEditingChanged];
-    self.arrayToDisplay = [PMCachingFunctions retreiveCache];
-    for (PMConversation *convo in self.arrayToDisplay) {
-        [self.convoTree addConversation:convo];
-    }
-    self.arrayToDisplay = [self.convoTree retreiveSubTree:kEmpt];
-    [self.tableView reloadData];
-    [self _runGetQuery];
+    
+    self.manager = [PMDataManager new];
```

Comment:
Let's make PMDataManager a singletons that way we can make sure we won't have multiple instances messing up with the cache file.